node/shard/src/lib.rs

```diff
@@ -117,9 +144,15 @@ impl ShardBlockChain {
             block_index: last_block.body.header.index + 1,
             shard_id: last_block.body.header.shard_id,
         };
+        let mut receipts = vec![];
+        for receipt_block in prev_receipts.iter() {
+            // TODO: check validity of the receipt block
+            // TODO: remove clone here
+            receipts.extend(receipt_block.receipts.clone());
```

Comment:
if you pass &[ReceiptBlock] then don't need to clone here.

Reply:
Fixed in 2e6dd8c